src/main/ngapp/accio/src/app/editor/mask.directive.ts

```diff
@@ -117,6 +129,14 @@ export class MaskDirective {
       // Fire event to draw pixel
       this.continuePaintEvent.emit(pixel);
     }
+
+    else if (this.tool == MaskTool.PAN && this.mouseDown) {
+      const offsetCoord = this.convertToUnscaledCoord(e.offsetX, e.offsetY);
+      this.newPanEvent.emit(
+        new Coordinate((offsetCoord[0] - this.coord[0] + this.translationCoords.x), 
+                       (offsetCoord[1] - this.coord[1] + this.translationCoords.y))
```

Comment:
the math here that you are converting coordinates would also make another good helper function like: convertToUnscaledCoord.